tests/slow/src/test/scala/tests/feature/CompletionCrossLspSuite.scala

```diff
@@ -5,9 +5,6 @@ import tests.BaseCompletionLspSuite
 
 object CompletionCrossLspSuite
     extends BaseCompletionLspSuite("completion-cross") {
-  testAsync("basic-211") {
```

Comment:
Can we reinstate this test?

Reply:
It’s fine if not, but I’m curious to know why we can’t keep the 2.11 tests.

Reply:
@olafurpg I initially removed these because after this PR we will not be running the cross Scala tests on JDK 8. The first version of Scala that supports class file parsing of JDK > 8 bytecode is 2.12.10. Notably, there is no 2.11.x version which supports this AFAIK. 

I can add them back in, and then we can either.

* Discriminate on the JDK version and not run the tests with 2.11.x if we are > 8
* Discriminate on the JDK version and even if it is > 8 only have it emit 8 compatible bytecode.

I'm on the fence about the utility of this myself.

Reply:
Nevermind, I realized just now that we get the discrimination for free as part of the `BaseCompletionLspSuite` changes. Sorry, I'm still learning my way around the code base.